src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -0,0 +1,349 @@
+                            <div className={styles.accordionChild}>
+                              <TitleSmall.Sans.Light300>
+                                <NavLink
+                                  href={link.href}
+                                  LinkComponent={
+                                    link.href !== '/login/'
+                                      ? LinkComponent
+                                      : null
+                                  }
+                                >
+                                  {link.title}
+                                </NavLink>
+                              </TitleSmall.Sans.Light300>
+                            </div>
+                          ))}
+                      </div>
+                    </div>
+                  ))}
+                </div>
+                <div class={styles.belowAccordion}>
+                  <div class={styles.belowAccordionItem}>
+                    <NavLink
+                      href={LINKS.TERM.href}
+                      LinkComponent={
+                        LINKS.TERM.href !== '/login/' ? LinkComponent : null
+                      }
+                    >
+                      <TitleMedium.Sans.Regular400>
+                        {LINKS.TERM.title}
+                      </TitleMedium.Sans.Regular400>
+                    </NavLink>
+                  </div>
+                  <div class={styles.belowAccordionItem}>
+                    <NavLink
+                      href={LINKS.ACCOUNT.href}
+                      LinkComponent={
+                        LINKS.ACCOUNT.href !== '/login/' ? LinkComponent : null
+                      }
+                    >
+                      <TitleMedium.Sans.Regular400>
+                        {LINKS.ACCOUNT.title}
+                      </TitleMedium.Sans.Regular400>
+                    </NavLink>
+                  </div>
+                  <div class={styles.belowAccordionItem}>
+                    <NavLink
+                      href={LINKS.SEARCH.href}
+                      LinkComponent={
+                        LINKS.SEARCH.href !== '/login/' ? LinkComponent : null
+                      }
+                    >
+                      <TitleMedium.Sans.Regular400>
+                        {LINKS.SEARCH.title}
+                      </TitleMedium.Sans.Regular400>
+                    </NavLink>
+                  </div>
+                </div>
+              </div>
+
+              {/* Mobile menu items, getAnEstimate only shows when scrolled */}
+              <NavLink
+                href={LINKS.INDEX.href}
+                LinkComponent={LinkComponent}
+                className={styles.phoneLogoFancy}
+              >
+                {FancyAnimatedLogo()}
+              </NavLink>
+              {!hideMobileCta && renderCtaButton(true)}
+              {renderSearchIcon(LINKS.SEARCH)}
+            </div>
+
+            <div className={styles.laptopAndUp}>
```

Comment:
Can we split out into a separate component

Reply:
should be able to

Reply:
marked in TODOs for fastfollow